judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/ContestRoleChecker.java

```diff
@@ -48,8 +48,10 @@ public ContestRole getRole(String userJid, Contest contest) {
             return ContestRole.MANAGER;
         } else if (canSupervise(userJid, contest)) {
             return ContestRole.SUPERVISOR;
+        } else if (contestRoleDao.isContestant(userJid, contest.getJid())) {
```

Comment:
Actually, let's just use `adminRoleDao` and `contestRoleDao` for all other if-checks above, because it's more efficient (no more redundant calls to the daos)